README.md

```diff
@@ -42,6 +42,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [F1Soft International](http://f1soft.com/) | Hattisar, Kathmandu |  FinTech company providing services to more than 90% of the banks and financial institutions in the country
 
 ## G - I
+- [Gurkhatech](https://gurkhatech.com/) | New Baneshwor, Kathmandu | App & Web Development, Facebook marketing Nepal
```

Comment:
`Gurkatech` should be below `Grepsr`. We are maintaining the list in alphabetical order. 

Can you also capitalize letter `m` of marketing to make your description consistent.